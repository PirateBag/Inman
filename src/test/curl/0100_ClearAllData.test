```Purpose
Clear the contents of the following tables:
  Item
  BOM
  orderLineItem
  Adjustment
```
==JSON==
{ "idToSearchFor" : "1" }

