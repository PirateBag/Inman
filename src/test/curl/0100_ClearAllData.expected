{
  "responseType" : null,
  "errors" : [ ],
  "data" : [ {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Items deleted"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "BOMs deleted"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "OrdereLineItems deleted"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Adjustments deleted"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Items PK Reset"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "BOMs PK Reset"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Order Line Items PK reset"
  }, {
    "id" : 0,
    "crudAction" : "NONE",
    "message" : "Adjustment PK reset"
  } ]
}